packages/x-date-pickers-pro/src/MobileDateTimeRangePicker/MobileDateTimeRangePicker.types.ts

```diff
@@ -0,0 +1,40 @@
+import { MakeOptional } from '@mui/x-date-pickers/internals';
+import {
+  UseMobileRangePickerSlotsComponent,
+  UseMobileRangePickerSlotsComponentsProps,
+  MobileRangeOnlyPickerProps,
+} from '../internals/hooks/useMobileRangePicker';
+import {
+  BaseDateTimeRangePickerProps,
+  BaseDateTimeRangePickerSlotsComponent,
+  BaseDateTimeRangePickerSlotsComponentsProps,
+} from '../DateTimeRangePicker/shared';
+import { DateTimeRangePickerView } from '../internals/models';
+
+export interface MobileDateTimeRangePickerSlotsComponent<TDate>
+  extends BaseDateTimeRangePickerSlotsComponent<TDate>,
+    MakeOptional<UseMobileRangePickerSlotsComponent<TDate, DateTimeRangePickerView>, 'field'> {}
+
+export interface MobileDateTimeRangePickerSlotsComponentsProps<TDate>
+  extends BaseDateTimeRangePickerSlotsComponentsProps<TDate>,
+    Omit<UseMobileRangePickerSlotsComponentsProps<TDate, DateTimeRangePickerView>, 'tabs'> {}
+
+export interface MobileDateTimeRangePickerProps<TDate>
+  extends BaseDateTimeRangePickerProps<TDate>,
+    MobileRangeOnlyPickerProps<TDate> {
+  /**
+   * The number of calendars to render on **Mobile**.
+   * @default 1
+   */
+  calendars?: 1 | 2 | 3;
```

Comment:
On `MobileDateRangePicker` we have the same prop but the prop description is wrong (it says "desktop").
On the component does not support custom `calendars`.

For `MobileDateTimeRangePicker` you are passing the `calendars` prop to the `DateRangeCalendar` component.
But the UI is totally broken.

IMHO we should just remove this prop from both `MobileDateTimeRangePicker` and `MobileDateRangePicker`

Reply:
I don't think it's worth treating it as a BC, because it already had no effect, so, it's just fixing the typing inconsistency. 🤔 
Please comment if you disagree. 😉

Reply:
Totally agree